src/leiningen/auto.clj

```diff
@@ -70,15 +70,16 @@
                       {:paths (default-paths project)}
                       (get-in project [:auto :default])
                       (get-in project [:auto task]))
-        directories (->> (mapcat directory-directories (:paths config))
-                          (seq))
-        fs (FileSystems/getDefault)
-        watcher (.newWatchService fs)
+        directories (mapcat subdirectories (:paths config))
+        watcher (.newWatchService (FileSystems/getDefault))
         keys (apply merge
               (for [dir directories]
                 (let [p (.toPath dir)
                       key (.register p watcher
-                        (into-array [StandardWatchEventKinds/ENTRY_CREATE StandardWatchEventKinds/ENTRY_MODIFY StandardWatchEventKinds/ENTRY_DELETE])
+                        (into-array [
+                          StandardWatchEventKinds/ENTRY_CREATE
```

Comment:
This needs to be on the previous line. Idiomatic Clojure indentation is more like:

```clojure
(into-array [StandardWatchEventKinds/ENTRY_CREATE
             StandardWatchEventKinds/ENTRY_MODIFY
             StandardWatchEventKinds/ENTRY_DELETE])
```